holidays/countries/india.py

```
@@ -527,8 +529,30 @@ def _populate_subdiv_hr_public_holidays(self):
+        # Bhagvan Shree Parshuram Jayanti
+        self._add_parshuram_jayanti(tr("Bhagvan Shree Parshuram Jayanti"))
+        # Maharana Pratap Jayanti.
+        self._add_maharana_pratap_jayanti(tr("Maharana Pratap Jayanti"))
+        # Sant Guru Kabir Jayanti (Prakat Diwas).
+        self._add_kabir_jayanti(tr("Sant Guru Kabir Jayanti (Prakat Diwas)"))
+        # Dussehra.
+        self._add_dussehra(tr("Dussehra"))
+        # Maharishi Valmiki's Birthday.
+        self._add_maharishi_valmiki_jayanti(tr("Maharishi Valmiki's Birthday"))
+
+    def _populate_subdiv_hp_women_optional_holidays(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_populate_subdiv_hp_women_optional_holidays`

Add return type annotation: `None`

(ANN202)